featomic/benches/lode-spherical-expansion.rs

```diff
@@ -1,24 +1,27 @@
 #![allow(clippy::needless_return)]
 use featomic::{Calculator, System, CalculationOptions};
+use chemfiles::{Frame, Trajectory};
 
 use criterion::{BenchmarkGroup, Criterion, measurement::WallTime, SamplingMode};
 use criterion::{criterion_group, criterion_main};
 
-fn load_systems(path: &str) -> Vec<Box<dyn System>> {
-    let systems = featomic::systems::read_from_file(format!("benches/data/{}", path))
-        .expect("failed to read file");
+fn read_systems_from_file(path: &str) -> Vec<Box<dyn System>> {
+    let mut trajectory = Trajectory::open(path, 'r').expect("could not open the trajectory");
```

Comment:
Does this really have to be mutable? You are just reading, right?

Reply:
Yes, it does because when reading a file we are modifying the internal data structure representing this file. See https://doc.rust-lang.org/std/fs/struct.File.html for example, where the `File` struct is mutable even when reading.